src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public Figure getRandomFigure() {
+        int index = new Random().nextInt(EnumFigure.values().length);
```

Comment:
random should be `private final` class-level field, because we want to create it's instance once and generate random numbers many times with it's help
```suggestion
        int index = random.nextInt(EnumFigure.values().length);
```